packages/core/src/awsService/appBuilder/activation.ts

```diff
@@ -200,6 +200,9 @@ async function registerAppBuilderCommands(context: ExtContext): Promise<void> {
                     await runSync('infra', arg, undefined, choices.syncParam)
                 }
             }
+        }),
+        Commands.register({ id: 'aws.toolkit.lambda.serverlessLand', autoconnect: false }, async () => {
```

Comment:
Would a more descriptive name be useful here? Without looking at the implementation I have no idea what this command does. Maybe `aws.toolkit.lambda.createServerlessLandProject`?